composer.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "jms/serializer",
     "type": "library",
-    "description": "Library for (de-)serializing data of any complexity; supports XML, JSON, and YAML.",
+    "description": "Library for (de-)serializing data of any complexity; supports XML, and JSON.",
```

Comment:
Hi @ @scyzoryck! 👋  I've seen this pull request from the [3.19.0](https://github.com/schmittjoh/serializer/releases/tag/3.19.0) release, maybe you can remove the unnecessary comma?

```diff
-     "description": "Library for (de-)serializing data of any complexity; supports XML, and JSON.",
+     "description": "Library for (de-)serializing data of any complexity; supports XML and JSON.",
```